packages/nango-yaml/lib/modelsParser.ts

```diff
@@ -0,0 +1,201 @@
+import type { NangoModel, NangoModelField, NangoYamlModel, NangoYamlModelFields } from '@nangohq/types';
+import { getNativeDataType, getPotentialTypeAlias, isDisallowedType, shouldQuote } from './helpers.js';
+import { ParserError, ParserErrorCycle, ParserErrorDataSyntax, ParserErrorExtendsNotFound, ParserErrorInvalidModelName } from './errors.js';
+
+export class ModelsParser {
```

Comment:
Small nit but I would keep this singular, `ModelParser`

Reply:
I see what you mean, it's like that because it's parsing the `models` list and contains all the models maybe ModelsPropParser would have been clearer